### ISSUE TITLE
Also process bounced unidentified message

### ISSUE THREAD

**Issue by lwcorp** [phpList/phplist3#1022]
After noticing my bounce count is roughly the same after there were supposed to be consecutive bounces, I've realized when bounces were signaled as `bounced unidentified message` - and in those cases it seems no processing takes place at all.
In other words, a subscriber's mail can bounce indefinitely and no action will be taken.

The reason is [function processMessages](https://github.com/phpList/phplist3/blob/main/public_html/lists/admin/processbounces.php#L409) checks `$GLOBALS['tables']['user_message_bounce']`, but that table only lists identified bounces, so it's kind of a vicious circle.

The irony is the user is known, it's just the message that is unknown. So if the user is known and we know there was a bounce, it should definitely go against that user's consecutive bounces and unconfirm if needed. Otherwise it affects being considered as a spammer.

**Comment by michield** [comment]
You mean the transactional emails, as opposed to a campaign? Is there any reason why the bounce is not identified? Maybe it's better to fix that instead.

**Comment by lwcorp** [comment]
No, I mean standard campaigns. I don't know why so many bounces are like this, but it's out of my control as it comes from various ISPs. All I know is phpList does know the user should it chooses to address it.

**Comment by HartLarsson** [comment]
> You mean the transactional emails, as opposed to a campaign? Is there any reason why the bounce is not identified? Maybe it's better to fix that instead.

sometimes i see mailbox have autoresponders that send the answer not to the sending mail but to the bounce mail and there is no headers to identify who answer and cannot be identified.

I think a good upgrade is to delete the bounce if is not identified (option that user can select somewhere in the config.php)
